packages/material-ui/src/Slide/Slide.js

```diff
@@ -152,13 +152,8 @@ const Slide = React.forwardRef(function Slide(props, ref) {
       },
     );
 
-    node.style.webkitTransition = theme.transitions.create('-webkit-transform', {
-      ...transitionProps,
-    });
-
-    node.style.transition = theme.transitions.create('transform', {
-      ...transitionProps,
-    });
+    node.style.webkitTransition = theme.transitions.create('-webkit-transform', transitionProps);
+    node.style.transition = theme.transitions.create('transform', transitionProps);
```

Comment:
Drive-by refactoring. `transitionProps` are not mutated by `theme.transitions.create`. That would be a bug.